third_party/openvino/ngraph_c_api/src/ngraph_c_api.cpp

```diff
@@ -119,6 +119,48 @@ inline ngraph::op::PadType GetAutoPad(ngraph_auto_pad autoPad) {
   return auto_pad;
 }
 
+inline std::string GetGruActivation(ngraph_gru_activation activation) {
```

Comment:
Will it bring build warning of uninitialized varibles?

Reply:
Removed this function.